packages/common/src/reducers/messages.js

```diff
@@ -263,12 +262,18 @@ export const selectMessage = createSelector(
     return selectMessage(state, message.reply_to);
   },
   (state) => state.user,
-  (message, author, repliedMessage, loggedInUser) => {
+  (state, messageId) => {
+    const message = state.messages.entriesById[messageId];
+    if (message == null || !message.app) return null;
+    return selectApp(state, message.app);
+  },
+  (message, author, repliedMessage, loggedInUser, app) => {
```

Comment:
I said boring Sunday, but Mondays can also be a bore 😂 

This was helpful to improve this piece:
https://redux.js.org/usage/deriving-data-selectors